include/zephyr/drivers/adc.h

```diff
@@ -681,6 +681,21 @@ static inline int z_impl_adc_read(const struct device *dev,
 	return api->read(dev, sequence);
 }
 
+/**
+ * @brief Set a read request from a struct adc_dt_spec.
+ *
+ * @param spec ADC specification from Devicetree.
+ * @param sequence  Structure specifying requested sequence of samplings.
+ *
+ * @return A value from adc_read().
+ * @see adc_read()
+ */
+static inline int adc_read_dt(const struct adc_dt_spec *spec,
+		       const struct adc_sequence *sequence)
```

Comment:
nit: could you check the indentation? don't think it's a hard rule but it looks like the rest of the file is indented on the parenthesis

Reply:
Thanks, fixed 👍